nemo/collections/asr/parts/utils/diarization_utils.py

```diff
@@ -686,104 +592,68 @@ def get_frame_level_VAD(self, oracle_model, audio_file_list):
             audio_file_path (List):
                 List contains file paths for audio files.
         """
-        for k, audio_file_path in enumerate(audio_file_list):
-            uniq_id = get_uniq_id_from_audio_path(audio_file_path)
-            vad_pred_diar = oracle_model.vad_pred_dir
-            frame_vad = os.path.join(vad_pred_diar, uniq_id + '.median')
-            frame_vad_list = read_file_paths(frame_vad)
-            frame_vad_float_list = [float(x) for x in frame_vad_list]
+        for uniq_id in self.AUDIO_RTTM_MAP:
+            frame_vad = os.path.join(vad_processing_dir, uniq_id + '.median')
```

Comment:
.median 
Will need to update the median here later. the ext could be either median or mean or frame as the user chose.